src/main/kotlin/no/nav/familie/ef/sak/beregning/BeregningBarnetilsynUtil.kt

```diff
@@ -0,0 +1,70 @@
+package no.nav.familie.ef.sak.beregning
+
+import no.nav.familie.ef.sak.beregning.barnetilsyndto.BeløpsperiodeBarnetilsynDto
+import no.nav.familie.ef.sak.beregning.barnetilsyndto.BeregningsgrunnlagBarnetilsynDto
+import no.nav.familie.ef.sak.felles.dto.Periode
+import java.math.BigDecimal
+import java.math.BigDecimal.ZERO
+import java.time.LocalDate
+import java.time.YearMonth
+
+data class MaxbeløpBarnetilsynSats(val fraOgMedDato: LocalDate,
+                                   val tilOgMedDato: LocalDate,
+                                   val maxbeløp: Map<Int, Int>)
+
+object BeregningBarnetilsynUtil {
+
+    val satserForBarnetilsyn: List<MaxbeløpBarnetilsynSats> =
+            listOf(MaxbeløpBarnetilsynSats(fraOgMedDato = LocalDate.parse("2022-01-01"),
```

Comment:
Litt forskjellig bruk av hvordan dato er i bruk er, kanskje vi burde forholde oss til en type? Eks `LocalDate.of(2022, 1, 1)`, i stedet for `LocalDate.parse("2022-01-01")`?

Reply:
Copy paste feil ;) , men jeg er enig - fikser.